phpBB/config/default/container/services.yml

```diff
@@ -116,6 +116,12 @@ services:
             - @controller.resolver
             - @request_stack
 
+    language:
```

Comment:
This service part should be rewritten, but I'm not sure how exactly.

Probably this part should be renamed to language.parent, made abstract, and then language service could link to it, and we could specify passing extension manager into the setter, but that should be environment dependent, so  I'm not sure where it should be specified.